snapcraft.yaml

```diff
@@ -11,7 +11,7 @@
 
 name: webots
 title: Webots
-base: core18
+base: core20
```

Comment:
This wasn't necessary, but I believe it is good to stay updated

Reply:
Based on the documentation (https://snapcraft.io/docs/base-snaps) `core18` is "the currently recommended base for the majority of snaps".